cypress/integration/chi-vue/search-input.spec.js

```diff
@@ -0,0 +1,216 @@
+const COMPONENT_CLASSES = {
+  CLOSE: '.-close',
+  SEARCH: '.-flat'
+}
+const EXAMPLE_EVENTS = {
+  CHI_BLUR: 'exampleChiBlur',
+  CHI_CHANGE: 'exampleChiChange',
+  CHI_CLEAN: 'exampleChiClean',
+  CHI_FOCUS: 'exampleChiFocus',
+  CHI_INPUT: 'exampleChiInput',
+  CHI_SEARCH: 'exampleChiSearch',
+}
+const EVENTS = {
+  BLUR: 'chiBlur',
+  CHANGE: 'chiChange',
+  CLEAN: 'chiClean',
+  FOCUS: 'chiFocus',
+  INPUT: 'chiInput',
+  SEARCH: 'chiSearch',
+}
+
+
+const checkSizeClass = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.class', value);
+};
+
+const checkAttr = (el, attr, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).invoke('attr', attr).should('eq', value);
+};
+
+const checkBooleanAttr = (el, attr, value) => {
+  cy.get(el).invoke('attr', attr).should('eq', value);
+};
+
+const checkValue = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.value', value)
+}
+
+describe('Search Input', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/search-input.html');
+  });
+
+  describe('Base', () => {
+    it('should exist base search input', () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.BASE}`)
+    });
+  });
+
+  describe('Disabled', () => {
+    it('should disable search input', () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DISABLED}`)
+        .find(`${ELEMENTS.INPUT}`)
+        .should('have.attr', 'disabled');
+    });
+  });
+
+  describe('Sizes', () => {
+    const sizes = ['sm', 'md', 'lg', 'xl'];
+
+    sizes.forEach((size) => {
+      it(`should have -${size} size`, () => {
+        checkSizeClass(`[data-cy='${size}-search-input-test']`, `-${size}`);
+      });
+    });
+  });
+
+  describe('Attrs', () => {
+    const attrs = ['name', 'placeholder'];
+    const booleanAttrs = ['portal', 'prevent-value-mutation']
+    const booleanValue = 'true';
+    const value = 'test';
+
+    attrs.forEach((attr) => {
+      it(`should have ${attr} attr`, () => {
+        checkAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, attr, value);
+      });
+    });
+
+    booleanAttrs.forEach((attr) => {
+      it(`should have ${attr} attr at true`, () => {
+        checkBooleanAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, attr, booleanValue);
+      });
+    });
+  });
+
+  describe('Check delete value', () => {
+    const value = 'test';
+    const emptyValue = '';
+
+    it(`should have value`, () => {
+      checkValue(`${SEARCH_INPUTS_DATA_CY.DELETE}`, value);
+    });
+
+    it(`should appear the icon to clean`, () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DELETE}`).find(`${COMPONENT_CLASSES.CLOSE}`);
+    });
+
+    it(`should click to clean`, () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DELETE}`).find(`${COMPONENT_CLASSES.CLOSE}`).click();
+    });
+
+    it(`should have not value`, () => {
+      checkValue(`${SEARCH_INPUTS_DATA_CY.DELETE}`, emptyValue);
+    });
+  });
+
+
+  describe('Events', () => {
+    it(`should trigger chiBlur`, () => {
+      cy.window()
+        .its(`${EXAMPLE_EVENTS.CHI_BLUR}`)
```

Comment:
```suggestion
        .its(`${EVENT_EXAMPLES.BLUR}`)
```
As mentioned above, the naming needs to be more descriptive. The constant does not contain events but references of event examples.

Reply:
Please,  update all the following cases as well

Reply:
Solved.